tardis/io/tests/test_model_reader.py

```diff
@@ -0,0 +1,35 @@
+import tardis
+from tardis.io.model_reader import (read_artis_density,
+read_simple_ascii_abundances)
+
```

Comment:
@aoifeboyle  I think its like this https://www.python.org/dev/peps/pep-0328/. @wkerzendorf ?